tests/test_config.py

```diff
@@ -38,10 +38,10 @@ def _compare_configs(self, dict_config, server_config):
         subclass thereof.
 
     """
-    cfg = vars(server_config).copy()
-    if 'version' in cfg:
-        cfg['version'] = str(cfg['version'])
-    self.assertEqual(dict_config, cfg)
+    if 'version' in dict_config:
+        dict_config = dict_config.copy()  # shadow the passed in dict
```

Comment:
Sorry, I misread the docstring and thought that you were missing a word for the 'what' :) ACK